shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -198,17 +209,16 @@ class JSBackend {
                   )
                 ) :: Nil
             case Def(isRecursive, name, R(body)) => Nil
-            case _: Term => Nil
+            case _: Term                         => Nil
           })
           // Generate something like `exprs.push(<expr>)`.
-          .concat(otherStmts.zipWithIndex.collect {
-            case (term: Term, index) =>
-              new JSExprStmt(
-                JSInvoke(
-                  JSMember(JSIdent(exprResultObjName), "push"),
-                  translateTerm(term)
-                )
+          .concat(otherStmts.zipWithIndex.collect { case (term: Term, index) =>
+            new JSExprStmt(
```

Comment:
Nitpick: from a code style point of view, it would be good to remove all these useless `new` keywords.

Reply:
Yes. They looks ugly. I was thinking about adding `apply` to all classes. Now it’s time to do it.

Reply:
You don't need to! The `apply` method is auto-generated for case classes!

Reply:
🥲 I thought I should implement them. Thanks for pointing out!